README.md

```diff
@@ -1 +1,64 @@
 # Inverted-Index-Project
+
+[![Code Climate](https://codeclimate.com/github/andela-fomokaro/Inverted-Index-Project/badges/gpa.svg?branch=develop)](https://codeclimate.com/github/andela-fomokaro/Inverted-Index-Project) [![Coverage Status](https://coveralls.io/repos/github/andela-fomokaro/Inverted-Index-Project/badge.svg?branch=develop)](https://coveralls.io/github/andela-fomokaro/Inverted-Index-Project?branch=develop)[![Build Status](https://travis-ci.org/andela-fomokaro/Inverted-Index-Project.svg?branch=develop)](https://travis-ci.org/andela-fomokaro/Inverted-Index-Project)
+
+
+
+## Introduction
+
+Inverted index takes a JSON array of text objects and creates an index from the array. The index allows a user to search for text blocks in the array that contain a specified collection of words.
+
+
+
+## References
+
+Inverted index - wikipedia.com
+
+Inverted index - elastic search
+
+
+## Limitations
+
+This app can only work for JSON files with members having title and text properties.
+
+## Running Locally
+
+Follow the steps below to run the application locally on your machine.
+
+Clone the repository: git clone <THIS_REPOSITORY_URL>
```

Comment:
Your steps to install the application locally are rather general and not specific. Consider replacing the generals with specifics, like `<THIS_REPOSITORY_URL>` with your repo link.

Also, it would appear more clearer if you put the steps in an unordered list